kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -553,7 +565,11 @@ define(['common/errorDisplay', 'common/format', 'common/html', 'common/props', '
             if (status === 'estimating' || status === 'created') {
                 status = 'queued';
             }
-            statuses[status] ? (statuses[status] += nJobs) : (statuses[status] = nJobs);
+            if (statuses[status]) {
```

Comment:
sonarlint doesn't like the ternary, even though it has been scientifically proven that ternaries are extremely awesome, especially where there is a bit of variable assignment going on in them.